controller/search_spaces_blackbox_test.go

```diff
@@ -29,6 +29,7 @@ type args struct {
 	pageOffset *string
 	pageLimit  *int
 	q          string
+	sort       *string
 }
 
```

Comment:
please add tests for sorting as well in this file
create few spaces using TestFixture like `tf.NewTestFixture(rest.T(), rest.DB, tf.Spaces(Count))` and try to list them with some sorting param. `_, list := test.SpacesSearchOK(..,.,.,..,.., "name")`

Reply:
added test to existing test method see here https://github.com/fabric8-services/fabric8-wit/pull/1887/files#diff-b531d7a8ab96948880ce591e9bac0fbbR84 is this fine?

Reply:
@surajssd  ah! sorry I missed it then. :+1:

Reply:
np @pranavgore09 :slightly_smiling_face: